spotbugs/src/main/java/edu/umd/cs/findbugs/detect/FindOverridableMethodCall.java

```diff
@@ -248,11 +248,9 @@ boolean checkDirectCase(XMethod caller, XMethod method, String message, int prio
     }
 
     private boolean shouldIgnoreCallInReadObject(XMethod method) {
-        // SER09-J-EX0: The readObject() method may invoke the overridable methods defaultReadObject() and readFields()
-        // in class java.io.ObjectInputStream
-        boolean inOIS = "java.io.ObjectInputStream".equals(method.getClassName());
-        String methodName = method.getName();
-        return inOIS && ("defaultReadObject".equals(methodName) || "readFields".equals(methodName));
+        // We're only interested in method calls on the object itself
+        // Calling ObjectInputStream.readInt() is not considered risky here because the object stream is potentially under the control of the attacker.
```

Comment:
I meant to say that this does not attempt to address the case when the attacker provides his own stream, because in that case all bets are off and the issue in that with overridable methods.
So, for instance, even though the `ObjectInputStream.readInt()` is overridable, calling it is acceptable. Using that input would be risky if it is user-supplied, but that beyond the scope of that detector.

Reply:
I think there is a negation missing. What do you think of this change?
```suggestion
        // Calling ObjectInputStream.readInt() is not considered risky here because the object stream is likely not under the control of the attacker.
```

Reply:
I've committed a proposed clarification, hopefully this makes more sense now!

Reply:
Perhaps "not considered risky here unless the object stream is already under the control of the attacker" and add a comment about that being out of scope for this bug.

Reply:
Can you please check the latest version? That's the message I tried to convey